### ISSUE TITLE
Feature: Full screen popup to avoid missing meetings

### ISSUE THREAD

**Issue by PDSSnyk** [leits/MeetingBar#572]
### Issue that this suggestion fixes

I find that even with the notifications, I am still ignoring them and turning up late to meetings.

### Solution description

It would be great if you could make a full-screen popup like in (https://www.inyourface.app/) so that missing meetings or joining them late is a thing of the past

### Alternatives considered

As mentioned above I have tried https://www.inyourface.app/ but it's not as good as your app.

**Comment by bigethan** [comment]
it might not work for you, but the auto join option solves this problem for me:
![Screen Shot 2023-01-11 at 9 21 15 AM](https://user-images.githubusercontent.com/627/211873720-5012defa-9f8f-43c9-8c0d-d31b3eb0180d.png)

**Comment by jasperrooduijn** [comment]
Any news on this! I would love to have a feature like this.

Thanks for the great work 👍

**Comment by iuri-gg** [comment]
This would be nice - the only feature missing for me. Just a popup dialog, not even fullscreen would do it. For now, I am using Applescript that it comes with by default:
```
on meetingStart(eventId, title, allday, startDate, endDate, eventLocation, repeatingEvent, attendeeCount, meetingUrl, meetingService, meetingNotes)
   tell application "Finder"
       activate
       display dialog title buttons {"OK"} default button "OK"
   end tell
end meetingStart
```

**Comment by leits** [comment]
@PDSSnyk, thanks for the request!
Yesterday, I released a new version with a minimal version of this feature. There is only a meeting title, service icon, time range, and button to join.

You can turn it on in the Advanced settings as an auto-join to the meeting. Please check it out!

**Comment by jasperrooduijn** [comment]
Very nice! Just tested this and it works as expected. Thanks for the good work.

**Comment by jasperrooduijn** [comment]
> @PDSSnyk, thanks for the request! Yesterday, I released a new version with a minimal version of this feature. There is only a meeting title, service icon, time range, and button to join.
> 
> You can turn it on in the Advanced settings as an auto-join to the meeting. Please check it out!

https://github.com/leits/MeetingBar/issues/572#issuecomment-1905973810

**Comment by leits** [comment]
"Show a fullscreen notification" is now available for all as a separate feature in 4.8.0 🚀

**Comment by jasperrooduijn** [comment]
>  "Show a fullscreen notification" is now available for all as a separate feature in 4.8.0 🚀 

Very nice! Good work 💪
